mllib/src/main/scala/org/apache/spark/mllib/linalg/BLAS.scala

```diff
@@ -257,30 +257,30 @@ private[spark] object BLAS extends Serializable with Logging {
 
   /**
    * C := alpha * A * B + beta * C
-   * @param transA whether to use the transpose of matrix A (true), or A itself (false).
-   * @param transB whether to use the transpose of matrix B (true), or B itself (false).
    * @param alpha a scalar to scale the multiplication A * B.
    * @param A the matrix A that will be left multiplied to B. Size of m x k.
    * @param B the matrix B that will be left multiplied by A. Size of k x n.
    * @param beta a scalar that can be used to scale matrix C.
-   * @param C the resulting matrix C. Size of m x n.
+   * @param C the resulting matrix C. Size of m x n. C.isTransposed must be false. In other words,
+   *          C cannot be the product of a `transpose()` call, or be converted from a transposed
```

Comment:
The last sentence is a little misleading. `C` could come from `transpose.transpose`. We can remove the last sentence completely. "C.isTransposed must be false.` is already sufficient.